app/src/renderer/components/common/TmSessionImport.vue

```diff
@@ -187,12 +184,8 @@ export default {
   },
   mounted() {
     this.$el.querySelector(`#import-name`).focus()
-    new PerfectScrollbar(this.$el.querySelector(`.tm-session-main`))
```

Comment:
lol why?

Reply:
it is not necessary anymore

Reply:
isn't it changing the look of the scrollbar on windows?

Reply:
hmm that i'm not sure of. but we can style scrollbars in css if need be. the use of this library seems like a code smell to me ... and predicting which component will need a custom scrollbar is not scalable.